rosbag2_compression/include/rosbag2_compression/base_compressor_interface.hpp

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ROSBAG2_COMPRESSION__BASE_COMPRESSOR_INTERFACE_HPP_
+#define ROSBAG2_COMPRESSION__BASE_COMPRESSOR_INTERFACE_HPP_
+
+#include <memory>
+#include <string>
+
+#include "rosbag2_storage/serialized_bag_message.hpp"
+#include "rosbag2_compression/visibility_control.hpp"
+
+typedef rosbag2_storage::SerializedBagMessage SerializedBagMessage;
```

Comment:
nit, the new syntax is more readable.

```suggestion
using SerializedBagMessage = rosbag2_storage::SerializedBagMessage;
```